src/attack.c

```diff
@@ -1074,8 +1074,18 @@ attack_network (struct scan_globals *globals)
   max_checks = get_max_checks_number ();
 
   hosts = gvm_hosts_new (hostlist);
-  unresolved = gvm_hosts_resolve (hosts);
+  if (hosts == NULL)
+    {
+      sprintf (buf, "Invalid target list: %s.", hostlist);
+      connect_main_kb (&main_kb);
+      message_to_client (main_kb, buf, NULL, NULL, "ERRMSG");
```

Comment:
`message_to_client` has internal buffer with len 2048. Make sure that no buffer overflow can happen in this function  too.